### PR TITLE
Fix fill_value handling bug in horizontal padding in torch backend's image.resize op

### LINKED ISSUE #22902: [Bug] Incorrect fill_value handling in Torch backend's image.resize with pad_to_aspect_ratio=True
When using keras.ops.image.resize with pad_to_aspect_ratio=True and a custom fill_value in the PyTorch backend, the left-side (horizontal) padding is hardcoded to 1.0 (white) instead of using the provided fill_value. This leads to incorrect numerical results when padding is applied horizontally.

Colab notebook reproducing this issue: https://colab.research.google.com/drive/1zjNW3WEUS5N6yNXRogjiW4Abg7qt4vBW?usp=sharing

In JAX:
```python
img = np.zeros((1, 10, 10, 3), dtype="float32")
resized = keras.ops.image.resize(img, (10, 20), pad_to_aspect_ratio=True, fill_value=0.5)
res_np = keras.ops.convert_to_numpy(resized)
```

gives
```output
Left padding pixel (index 0):  [1. 1. 1.]
Right padding pixel (index 19): [0.5 0.5 0.5]
```

while other backends give:
```output
Left padding pixel (index 0):  [0.5 0.5 0.5]
Right padding pixel (index 19): [0.5 0.5 0.5]
```
which is the correct output as the fill value is set to 0.5 by the user.

### PR DESCRIPTION
**Pull request by buildwithsuhana**
Fixes https://github.com/keras-team/keras/issues/22902

This PR fixes a numerical bug in the PyTorch backend where keras.ops.image.resize ignored the user-specified fill_value for horizontal left-side padding when pad_to_aspect_ratio=True. In the PyTorch backend's image.resize implementation, the left-side horizontal padding was hardcoded to 1.0 (white) because the fill_value multiplier was missing from the tensor operation. This resulted in incorrect padding colors whenever horizontal padding was required.

## Contributor Agreement

**Please review our [AI-Assisted Contribution Policy](../CONTRIBUTING.md#ai-assisted-contribution-policy) and check all boxes below before submitting your PR for review:**

- [x] I am a human, and not a bot.
- [x] I will be responsible for responding to review comments in a timely manner.
- [x] I will work with the maintainers to push this PR forward until submission.

**Note:** Failing to adhere to this agreement may result in your future PRs no longer being reviewed.

### CODE REVIEW

**Review by hertschuh — APPROVED** [review]
Thanks for the fix!
